File: idea-plugin/src/main/java/ch/yvu/teststore/idea/plugin/dto/RunDto.java
package ch.yvu.teststore.idea.plugin.dto;

import java.time.LocalDateTime;

public class RunDto {
	private String id;
	private String revision;
	private long time;

	public String getId() {
		return id;
	}

	public void setId(String id) {
		this.id = id;
	}

	public String getRevision() {
		return revision;
	}

	public void setRevision(String revision) {
		this.revision = revision;
	}

	public long getTime() {
		return time;
	}

	public void setTime(long time) {
		this.time = time;
	}
}

File: backend/src/main/kotlin/ch/yvu/teststore/insert/dto/RunDto.kt
package ch.yvu.teststore.insert.dto

import com.fasterxml.jackson.annotation.JsonCreator
import com.fasterxml.jackson.annotation.JsonProperty
import java.util.*

data class RunDto @JsonCreator constructor(
        @JsonProperty("revision") val revision: String? = null,
        @JsonProperty("time") val time: Date,
        @JsonProperty("tags") val tags: Map<String, String>)